ci-operator/step-registry/rosa/cluster/upgrade/cluster/rosa-cluster-upgrade-cluster-commands.sh

```
@@ -353,13 +354,23 @@ if [[ "$end_version_x" == "$current_version_x" ]]; then
   fi
 
   for y in $(seq $start_version_y $end_version_y); do
+    if [[ -n "${UPGRADE_CHANNEL}" && $y -ne $current_version_y ]]; then
+      log "Changing cluster channel to ${UPGRADE_CHANNEL} for Y-stream upgrade"
+      rosa edit cluster -c $cluster_id --channel "${UPGRADE_CHANNEL}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)